openedx/features/course_experience/utils.py

```diff
@@ -192,6 +207,7 @@ def recurse_mark_auth_denial(block):
     if course_outline_root_block:
         populate_children(course_outline_root_block, all_blocks['blocks'])
         recurse_mark_scored(course_outline_root_block)
+        recurse_num_graded_problems(course_outline_root_block)
         recurse_mark_auth_denial(course_outline_root_block)
```

Comment:
I'm wondering if we have enough things recursing up and down the tree here that this should just be a separate block transformer that does these accumulations. This was the problem they were intended to solve.

@ormsbee, care to weigh in, since you have a bit more backrground on this part of the stack?